Description: Upgrade to puppet_forge 2
Author: Michael Moll <Michael Moll kvedulv@kvedulv.de>
Origin: upstream, https://github.com/rodjek/librarian-puppet/pull/331
Forwarded: not-needed
Last-Update: 2016-03-11
Reviewed-by: Sebastien Badia <seb@sebian.fr>

--- a/lib/librarian/puppet/source/forge/repo_v3.rb
+++ b/lib/librarian/puppet/source/forge/repo_v3.rb
@@ -22,7 +22,7 @@ module Librarian
           end
 
           def dependencies(version)
-            array = get_release(version).metadata[:dependencies].map{|d| [d['name'], d['version_requirement']]}
+            array = get_release(version).metadata[:dependencies].map{|d| [d[:name], d[:version_requirement]]}
             Hash[*array.flatten(1)]
           end
 
@@ -32,7 +32,7 @@ module Librarian
             else
               # should never get here as we use one repo object for each module (to be changed in the future)
               debug { "Looking up url for #{name}@#{version}" }
-              release = PuppetForge::Release.find("#{name}-#{version}")
+              release = PuppetForge::V3::Release.find("#{name}-#{version}")
             end
             "#{source}#{release.file_uri}"
           end
@@ -40,8 +40,11 @@ module Librarian
         private
 
           def get_module
-            @module ||= PuppetForge::Module.find(name)
-            raise(Error, "Unable to find module '#{name}' on #{source}") unless @module
+            begin
+              @module ||= PuppetForge::V3::Module.find(name)
+            rescue Faraday::ResourceNotFound => e
+              raise(Error, "Unable to find module '#{name}' on #{source}")
+            end
             @module
           end
 
--- a/metadata.yml
+++ b/metadata.yml
@@ -45,14 +45,14 @@ dependencies:
     requirements:
     - - "~>"
       - !ruby/object:Gem::Version
-        version: '1.0'
+        version: '2.1'
   type: :runtime
   prerelease: false
   version_requirements: !ruby/object:Gem::Requirement
     requirements:
     - - "~>"
       - !ruby/object:Gem::Version
-        version: '1.0'
+        version: '2.1'
 - !ruby/object:Gem::Dependency
   name: rake
   requirement: !ruby/object:Gem::Requirement
